src/icons/icons.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { select, number, text } from '@storybook/addon-knobs';
+import ItemIcon, { ItemIconProps } from './item-icon';
+import ItemIconMonotone from './item-icon-monotone';
+import Tooltip, { TooltipPosition } from '../components/tooltip';
+
+const itemTypeOptions: ItemIconProps['iconType'][] = [
+    'default',
+    'audio',
+    'bookmark',
+    'boxnote',
+    'code',
+    'document',
+    'dwg',
+    'excel-spreadsheet',
+    'google-docs',
+    'google-sheets',
+    'google-slides',
+    'illustrator',
+    'image',
+    'indesign',
+    'keynote',
+    'numbers',
+    'pages',
+    'pdf',
+    'photoshop',
+    'powerpoint-presentation',
+    'presentation',
+    'spreadsheet',
+    'text',
+    'threed',
+    'vector',
+    'video',
+    'word-document',
+    'zip',
+    'folder-collab',
+    'folder-external',
+    'folder-plain',
```

Comment:
Seems weird that the folder types alone are not listed in alphabetical order

Reply:
I put the folders at the end and "default" icon at the beginning instead of strict alphabet order to make it find them instead of getting lost in the filetype list